webapp/src/Controller/API/JudgehostController.php

```diff
@@ -900,6 +910,7 @@ private function addSingleJudgingRun(
         string $outputError,
         string $outputDiff,
         string $outputRun,
+        string $teamMessage,
```

Comment:
```suggestion
        ?string $teamMessage,
```

Then it also accepts null and you don't need any casting.